cli/chaosli/cmd/root.go

```diff
@@ -86,34 +89,48 @@ func initConfig() {
 }
 
 func DisruptionFromFile(path string) (v1beta1.Disruption, error) {
-	yaml, err := os.Open(filepath.Clean(path))
+	parsedSpec := ReadUnmarshalValidate(path)
+
+	return parsedSpec, nil
+}
+
+func PrintSeparator() {
+	fmt.Println("=======================================================================================================================================")
+	// after a separator we should assume there is information we want the user to read before more prints start
+	// flooding the terminal. That is why we add this sleep of 2 seconds to give some time for the user to
+	// digest the new information before consuming the next
+	time.Sleep(2 * time.Second)
+}
+
+func ReadUnmarshalValidate(path string) v1beta1.Disruption {
+	fullPath, err := filepath.Abs(path)
+	if err != nil {
+		log.Fatalf("finding Absolute Path: %v", err)
+	}
+
+	yaml, err := os.Open(filepath.Clean(fullPath))
 	if err != nil {
-		return v1beta1.Disruption{}, err
+		log.Fatalf("could not open yaml: %v", err)
 	}
 
 	yamlBytes, err := ioutil.ReadAll(yaml)
 	if err != nil {
-		return v1beta1.Disruption{}, err
+		log.Printf("disruption.Get err   #%v ", err)
+		os.Exit(1)
```

Comment:
Why not use `log.Fatal` here as well? Is the exitcode for a fatal log not `1`?

Reply:
log.Fatal should be fine here and good for consistency as well.